src/base/noscript.scss

```diff
@@ -8,6 +8,12 @@
 	display: block !important;
 }
 
+@media screen {
+	%noscript-screen-display-block-important {
```

Comment:
When I first sent this PR, my goal was to change ``%noscript-display-block-important`` to only apply to screen viewports in a handful of scenarios (not all). That's why I created a second variant of it instead of completely replacing it.

But given your feedback and my other comment, I'll go through with your suggestion and stick with the new placeholder selector's name (``noscript-screen-display-block-important``), since it clearly denotes that it only applies to screen viewports.